injector/cpu_pressure.go

```diff
@@ -26,13 +26,14 @@ type cpuPressureInjector struct {
 // CPUPressureInjectorConfig is the CPU pressure injector config
 type CPUPressureInjectorConfig struct {
 	Config
-	Stresser       stress.Stresser
-	StresserExit   chan struct{}
-	ProcessManager process.Manager
+	Stresser        stress.Stresser
+	StresserExit    chan struct{}
+	ProcessManager  process.Manager
+	StresserManager StresserManager
 }
 
 // NewCPUPressureInjector creates a CPU pressure injector with the given config
-func NewCPUPressureInjector(spec v1beta1.CPUPressureSpec, config CPUPressureInjectorConfig) Injector {
+func NewCPUPressureInjector(spec v1beta1.CPUPressureSpec, config CPUPressureInjectorConfig, stresserManager StresserManager) Injector {
```

Comment:
As it's part of `CPUPressureInjectorConfig` already provided as an argument, why adding a new argument?

You may want to change the `NewCPUPressureInjector` signature to return an error if it's empty, as it's required for most of this struct methods.

```suggestion
func NewCPUPressureInjector(spec v1beta1.CPUPressureSpec, config CPUPressureInjectorConfig) (Injector, error) {
```

Reply:
yeah, good call out. Thanks